app/views/organisations/users.html.erb

```diff
@@ -12,10 +12,12 @@
   <h2 class="govuk-visually-hidden">Users</h2>
 <% end %>
 
-<% if current_user.data_coordinator? || current_user.support? %>
-  <%= govuk_button_link_to "Invite user", new_user_path(organisation_id: @organisation.id), html: { method: :get } %>
-<% end %>
 <div class="app-filter-layout" data-controller="filter-layout">
+  <div class="govuk-button-group app-filter-toggle govuk-!-margin-bottom-6">
+    <% if current_user.data_coordinator? || current_user.support? %>
+      <%= govuk_button_link_to "Invite user", new_user_path(organisation_id: @organisation.id), html: { method: :get } %>
+    <% end %>
+  </div>
```

Comment:
The key change was to include the class `app-filter-toggle`. It could be added to the button group like this or separately with `<div class="app-filter-toggle"><div>` (Although this will not expand on even smaller screen and fill the width like the other buttons without further work). Either way, it must be contained within a `div` with the class `app-filter-layout`.